src/core/Kokkos_RemoteSpaces_ViewMapping.hpp

```diff
@@ -1157,7 +1157,8 @@ class ViewMapping<Traits, Kokkos::Experimental::RemoteSpaceSpecializeTag> {
   template <class... P, typename T = Traits>
   Kokkos::Impl::SharedAllocationRecord<> *allocate_shared(
       Kokkos::Impl::ViewCtorProp<P...> const &arg_prop,
-      typename Traits::array_layout const &arg_layout) {
+      typename Traits::array_layout const &arg_layout,
+      bool execution_space_specified) {
```

Comment:
make sure you mention that this change broke remote spaces too at the kokkos dev meeting